src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/address/Address.kt

```diff
@@ -18,43 +15,40 @@ enum class AddressSource {
 
 @Entity
 @Table(name = "address")
-data class Address(
-
-  @Id
-  @GeneratedValue(strategy = GenerationType.IDENTITY)
-  @param:Positive
-  val id: Long? = null,
+class Address(
+  id: Long? = null,
```

Comment:
changed this from data class to normal entity to resolve issues with lazy loading,.. notice the update and create code do not need save the entity now,.. it all happens on close of transaction.